app/permissions_migrations.go

```diff
@@ -56,6 +57,9 @@ const (
 	PERMISSION_INVITE_GUEST                      = "invite_guest"
 	PERMISSION_PROMOTE_GUEST                     = "promote_guest"
 	PERMISSION_DEMOTE_TO_GUEST                   = "demote_to_guest"
+	PERMISSION_CHANNEL_MENTION                   = "channel_mention"
```

Comment:
All of the other permissions contain a verb but this one does not. I'm wondering if `use_channel_mentions` is more idiomatic. I suppose the argument could be made that "mention" is the verb, and in that case would `mention_channel` be better?

Reply:
I think `mention_channel` makes the most sense - like you said `mention` is the verb like `delete_channel` or `create_channel`.

Reply:
actually I just got went through the tech spec again and it actually has the permission down as `use_channel_mentions` so ill go with that.

Reply:
@fm2munsh Oops, I just changed it. It's your call I can update the spec to fit your decision.

Reply:
haha I just modified my code to `use_channel_mentions` so lets revert it back to that if that still works for you 🙂

Reply:
@fm2munsh Yeah sounds good; will do 👍